tests/snuba/api/endpoints/test_project_group_index.py

```diff
@@ -352,6 +352,36 @@ def test_filter_not_unresolved(self):
         assert response.status_code == 200
         assert [int(r["id"]) for r in response.data] == [event.group.id]
 
+    def test_group_by_hash(self):
+        event = self.store_event(
+            data={"timestamp": iso_format(before_now(seconds=500)), "fingerprint": ["group-1"]},
+            project_id=self.project.id,
+        )
+
+        self.login_as(user=self.user)
+
+        response = self.client.get(f"{self.path}?hash={event.get_primary_hash()}")
+        assert response.status_code == 200
+        assert len(response.data) == 1
```

Comment:
Maybe we should test that it sends back the correct group here also. Don't have to assert on the whole serialized response, but you could just grab the id and test against that. WDYT?

Reply:
yeah, that makes sense!